clients/stellar-relay-lib/resources/config/testnet/stellar_relay_config_sdftest1.json

```diff
@@ -10,5 +10,7 @@
     "version_str": "stellar-core 20.2.0.rc3 (34d82fc00426643e16b7ad59c9fde169b778eb4b)",
     "is_pub_net": false
   },
-  "stellar_history_archive_urls": []
+  "stellar_history_archive_urls": [
+    "http://history.stellar.org/prd/core-testnet/core_testnet_001"
```

Comment:
any reason why we don't include `core_testnet_002` and `core_testnet_003`?

Reply:
No particular reason, in fact I thought about adding all of them, but then looking at the analogous `stellar_relay_config_mainnet_iowa.json` I see that it only uses [one](https://github.com/pendulum-chain/spacewalk/blob/189765ca732396900fb4b49caa1ab6e82c7a9180/clients/stellar-relay-lib/resources/config/mainnet/stellar_relay_config_mainnet_iowa.json#L14) of the mainnet archives. 

You think every config file should have all the possible archive url's @b-yap, including for mainnet?

Reply:
I guess the only argument in favor of separating them could be to have a more fine-grained control in case one of them is not working, so that the selection is not random when trying to fetch.

Reply:
I think this is just some legacy fragment, because in the very beginning we only knew about one Stellar archive URL. By now, we know 'all' of the official ones and there is no good reason not to add all of them to the array. 

It might be least confusing if we streamline this across all the configuration files and make each of them list all archives (for the respective network only, of course).

Reply:
Alright, also makes sense. Added [here](https://github.com/pendulum-chain/spacewalk/pull/524/commits/af6cb702eb0a903253f8cb0f4035ada4de767a08).
We could also add `stellar-history` as well as `core-live` but seems to me a bit confusing.

Reply:
Hmm.. let's maybe still have all 6 in the mainnet configurations to signify that all of them can be used.